plugins/woocommerce/includes/class-wc-checkout.php

```diff
@@ -924,7 +924,7 @@ protected function validate_checkout( &$data, &$errors ) {
 
 		// phpcs:ignore WordPress.Security.NonceVerification.Missing
 		if ( empty( $data['woocommerce_checkout_update_totals'] ) && empty( $data['terms'] ) && ! empty( $data['terms-field'] ) ) {
-			$errors->add( 'terms', __( 'Please read and accept the terms and conditions to proceed with your order.', 'woocommerce' ) );
+			$errors->add( 'terms', __( 'Please read and accept the terms and conditions to proceed with your order.', 'woocommerce' ), array( 'id' => 'terms') );
```

Comment:
Add the `id` attribute to the "terms" error message so it can be associated with the input.